### ISSUE TITLE
Flow: Input type sets fields as required

### ISSUE THREAD

**Issue by robvk** [dotansimha/graphql-code-generator#1645]
**Describe the bug**
When I use the input keyword all fields are marked as required (so `email: ?string` instead of `email?: ?string`). 

**To Reproduce**

1. My GraphQL schema:

```graphql
schema {
   mutation: Mutation
}

type Mutation {
  updateUser(userId: ID!, update: UpdateUser): User!
}

type User {
  id: ID!
  username: String!
  email: String!
}

input UpdateUser {
  username: String,
  email: String
}
```

3. My `codegen.yml` config file:

```yml
generates:
  types.flow.js:
    - flow
    
```

This gives the following *flow* output for UpdateUser:
```
export type UpdateUser = {
  username: ?$ElementType<Scalars, 'String'>,
  email: ?$ElementType<Scalars, 'String'>,
};
```

**Expected behavior**
The output I would expect is:

```
export type UpdateUser = {
  username?: ?$ElementType<Scalars, 'String'>,
  email?: ?$ElementType<Scalars, 'String'>,
};
```

**Environment:**
It is replayable on the try it out section on the site.

**Context:**
If I try the same schema using typescript it does give the correct typing:
```
export type UpdateUser = {
  username?: Maybe<Scalars['String']>,
  email?: Maybe<Scalars['String']>,
}
```

It also seems to be specific to the `input` keyword. When using the `type` keyword it does convert as expected.

**Comment by dotansimha** [comment]
Thanks @robvk , 
Fixed it in: https://github.com/dotansimha/graphql-code-generator/pull/1663

**Comment by dotansimha** [comment]
Fixed in 1.0.7 :)
